packages/block-library/src/query/editor.scss

```diff
@@ -49,15 +49,6 @@
 	}
 }
 
-.block-library-query-toolspanel__filters {
```

Comment:
We're still rendering the original classname for backward compatibility, right?

Reply:
Correct. I added a code comment b912e8b